ssas/service/admin/api_test.go

```diff
@@ -25,6 +65,15 @@ func (s *APITestSuite) TearDownSuite() {
 	ssas.Close(s.db)
 }
 
+func (s *APITestSuite) TestCreateGroup() {
+	req := httptest.NewRequest("POST", "/group", strings.NewReader(sampleGroup))
+	handler := http.HandlerFunc(createGroup)
+	rr := httptest.NewRecorder()
+	handler.ServeHTTP(rr, req)
```

Comment:
Should we delete the test group at the end of the test?  If so, consider using `ssas.CleanDatabase(group)`.